app/views/modals/create-secret.html

```diff
@@ -0,0 +1,22 @@
+<div class="create-secret-modal">
+  <div class="modal-header">
+    <h2>
+      Create {{type | capitalize}} Secret
+      <a href="{{'creating_secrets' | helpLink}}" aria-hidden="true" target="_blank"><span class="learn-more-inline">Learn more<i class="fa fa-external-link"></i></span></a>
```

Comment:
`ng-href` and remove `aria-hidden="true"` from the link. `aria-hidden="true"` should be on the external icon, though.